.github/workflows/ci.yml

```diff
@@ -8,6 +8,9 @@ jobs:
   test:
     name: Test
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        product: [release, eap]
```

Comment:
`product` is a bit confusing for me. I would think that is `IntelliJ` or `GoLand`
Maybe `release_type` to be consistent with https://data.services.jetbrains.com/products?fields=code,name,releases.downloads,releases.version,releases.build,releases.type&code=IIC,IIE,GO ?

Reply:
Sure.